src/doc/about.rst

```diff
@@ -0,0 +1,156 @@
+=======================
+About the Kaldi project
+=======================
+
+---------------
+ What is Kaldi?
+---------------
+
+Kaldi is a toolkit for speech recognition written in C++ and licensed under
+the  Apache License v2.0.  Kaldi is intended for use by speech
+recognition researchers.  For more detailed history and list of contributors see
+\ref history.
```

Comment:
I am afraid that `*.rst` files converted from `*.dox` files have incorrect syntax of cross-referencing.